lib/gs-plugin-loader.c

```diff
@@ -3177,6 +3178,11 @@ gs_plugin_loader_process_thread_cb (GTask *task,
 		}
 	}
 
+	if (action == GS_PLUGIN_ACTION_UPGRADE_TRIGGER) {
+		GsPluginLoaderPrivate *priv = gs_plugin_loader_get_instance_private (plugin_loader);
```

Comment:
Nitpick: Usually you'd want to put the `priv` in the outter scope because it'll often end up being used in other places of the function, but I guess that for now we can keep it close to where it's used.